web/src/lib/domain-logic/getContentWithCardCount.ts

```diff
@@ -0,0 +1,30 @@
+import { filterCertifications } from "@/lib/domain-logic/filterCertifications";
+import { filterFundings } from "@/lib/domain-logic/filterFundings";
+import { getVisibleCertifications } from "@/lib/domain-logic/getVisibleCertifications";
+import { getVisibleFundings } from "@/lib/domain-logic/getVisibleFundings";
+import { Certification, Funding } from "@/lib/types/types";
+import { LookupOperatingPhaseById } from "@businessnjgovnavigator/shared/operatingPhase";
+import { Business } from "@businessnjgovnavigator/shared/userData";
+
+export const getContentWithCardCount = (
+  business: Business | undefined,
+  certifications: Certification[],
+  fundings: Funding[]
+): number => {
+  let count = 0;
+  if (LookupOperatingPhaseById(business?.profileData.operatingPhase).displayCertifications) {
+    count += getVisibleCertifications(
+      filterCertifications(certifications, business as Business),
```

Comment:
[boulder] All over this function, we are writing `business as Business` which is a big anti-pattern. 

The function has explicitly defined that the `business` argument has type `Business | undefined`. By using `as Business` throughout, we're basically losing any type safety this function might have had. And the type safety exists for a reason. `business` _could_ be undefined, and if it is, then what happens? Does the code break at runtime in prod with a "cannot read properties of undefined" error?

Our code should always be type-safe, and that includes handling a possible undefined gracefully and not forcing it to be ignored using `as`. There's lots of approaches here. I'd recommend a guard clause because that's the pattern we use in the rest of the code. What _should_ this return when `business` is undefined? Let's code (and even test!) for that possible use case.

Reply:
put a guard clause in to return 0 if its undefined.